### DISCUSSION TITLE
Why isn't there a "large-v3" option in the Whisper model list?

### DISCUSSION THREAD

**Discussion by jasonsu123** [openai/whisper#2096] [General]
Hello everyone, I've been rewriting the Whisper code recently because it seems that openai.Audio.transcribe is no longer supported.

The original code result = openai.Audio.transcribe("whisper-1", f) was changed to model = whisper.load_model(" ").

However, the response only lists the available models as ['tiny.en', 'tiny', 'base.en', 'base', 'small.en', 'small', 'medium.en', 'medium', 'large-v1', 'large-v2', 'large'].

Why is there no "large-v3" option? 

Thank you
          Best regards;

**Comment by Purfview** [comment]
You are using old version, update whisper to the latest release:
`pip install -U openai-whisper`

**Reply by jasonsu123** [reply]
Here's the translation of your text into English:

Thank you very much for your response. After updating, I do have the 'large-v3' option available. However, I'd like to know how to write the new version of the OpenAI Whisper code.

Below is my current Whisper code. Could you help me see if there are any issues with it?

My computer can only run on a CPU, and the whisper.load_model currently can only execute the 'base' size. If I try running 'large', it seems to crash.
However, strangely, after running the code, my OpenAI usage amount has not changed, whereas previously, when I used the 'whisper-1' model for processing, there would be billing involved.

```
audio_file = AudioSegment.from_file("test.mp3")

chunk_size = 60 * 1000

chunks = [audio_file[i:i+chunk_size] for i in range(0, len(audio_file), chunk_size)]

openai.api_key = "XXXXXXXXXXXXXXXX"

transcript = ""

model = whisper.load_model("large")

for i, chunk in enumerate(chunks, start=1):
    output_filename = f"temp_{i}.wav"
    chunk.export(output_filename, format="wav")
    result = model.transcribe(output_filename)
    raw_text = result["text"]
    print(f"Chunk {i}: {raw_text}")
    transcript += raw_text + '\n'
transcript = transcript.replace('\n\n', '\n')
print("Final: " + transcript)
```
